### PR TITLE
analytics style tweaks

### PR DESCRIPTION
**Pull request by adorsk**
#### What are the relevant tickets?
Partially implements #603 . Width fix still in progress.

#### What's this PR do?
1. Moves time/view counts table above graph.
2. Increases size of y-axis labels
3. Adds 'time' label to x-axis
4. Puts channel color in front of channel name
5. Makes progress slider more prominent

#### How should this be manually tested?
1. Open the analytics pane for a video that has analytics.
2. Ensure that it looks reasonable (except for issues of width relative to video)

### CODE REVIEW

**Comment by noisecapella** [comment]
Are there instructions on how to generate fake analytics?

**Comment by adorsk** [comment]
@noisecappela
Ah, sorry, I should have included instructions for mocking analytics data.

One way to mock analytics data is to set `window.ovsMockAnalyticsData`, per [lib/api.js#93](https://github.com/mitodl/odl-video-service/blob/9efa1a25307392f872c7a70a4ddf4e6dacc94ca8/static/js/lib/api.js#L93):
1. Go to a video detail page.
2. In the devtools js console, enter something like this:
```
let makeVideoAnalyticsData = (n, multichannel) => {
  n = n || 24
  const videoAnalyticsData = {
    is_multichannel: multichannel,
    times:           [...Array(n).keys()],
    channels:        multichannel
      ? [...Array(4).keys()].map(i => `channel${i + 1}`)
      : ["views"],
    views_at_times: {}
  }
  for (let tIdx = 0; tIdx < videoAnalyticsData.times.length; tIdx++) {
    const t = videoAnalyticsData.times[tIdx]
    const viewsAtTime = {}
    for (let cIdx = 0; cIdx < videoAnalyticsData.channels.length; cIdx++) {
      const channel = videoAnalyticsData.channels[cIdx]
      viewsAtTime[channel] =
        videoAnalyticsData.times.length - Math.floor((tIdx + 1) / (cIdx + 1))
    }
    videoAnalyticsData.views_at_times[t] = viewsAtTime
  }
  return videoAnalyticsData
}
window.ovsMockAnalyticsData = makeVideoAnalyticsData(120, true)
```
3. Open the analytics overlay.

File: static/js/components/analytics/AnalyticsChart.js [hunk 1]
```diff
@@ -135,14 +146,12 @@ export class AnalyticsChart extends React.Component {
           dependentAxis
           label="views"
           offsetX={padding.left - 2}
+          axisLabelComponent={
+            <VictoryLabel dy={-1 * (baseAxisLabelStyle.fontSize + 5)} />
```
Comment (noisecapella) [inline static/js/components/analytics/AnalyticsChart.js]: Is `fontSize` a pixel value here?

File: static/js/components/analytics/AnalyticsChart.js [hunk 2]
```diff
@@ -231,6 +233,31 @@ export class AnalyticsChart extends React.Component {
...
+          analyticsData.channels.map(channel => {
+            return viewsAtTime[channel] || 0
+          })
+        )
+      })
+      .sort()
+    const ticks = [0]
+    if (sortedTotalViewsValues.length > 0) {
+      if (sortedTotalViewsValues.length > 1) {
+        ticks.push(Math.round(_.mean(sortedTotalViewsValues)))
```
Comment (noisecapella) [inline static/js/components/analytics/AnalyticsChart.js]: Just to be sure, is the expected y tick values always supposed to be minimum, mean, maximum?

File: static/js/components/analytics/AnalyticsChart.js (hunk 1 above)
Comment (adorsk) [reply static/js/components/analytics/AnalyticsChart.js]: this `fontSize` is a value in terms of SVG units

File: static/js/components/analytics/AnalyticsChart.js (hunk 2 above)
Comment (adorsk) [reply static/js/components/analytics/AnalyticsChart.js]: Yes, that is the expectation.

**Pull request merged**